From 93a3be046b4ff2a9a04dd4f0f85222b656b0bc05 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Mon, 15 Dec 2014 08:19:16 -0500
Subject: [PATCH] Add FreeBSD support for PandaBoard

---
 include/configs/omap4_panda.h | 38 ++++++++++++++++++++++++++++++++++++++
 1 file changed, 38 insertions(+)

diff --git a/include/configs/omap4_panda.h b/include/configs/omap4_panda.h
index 7378acd..3f7cea3 100644
--- a/include/configs/omap4_panda.h
+++ b/include/configs/omap4_panda.h
@@ -47,4 +47,42 @@
 
 #define CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG
 
+#undef CONFIG_SYS_THUMB_BUILD
+#define CONFIG_SYS_ICACHE_OFF
+#define CONFIG_SYS_DCACHE_OFF
+
+#ifdef CONFIG_SPL_BUILD
+
+#undef CONFIG_CMD_PING
+#undef CONFIG_CMD_DHCP
+#undef CONFIG_CMD_NET
+
+#undef CONFIG_I2C
+#undef CONFIG_CMD_I2C
+#undef CONFIG_SYS_I2C
+#undef CONFIG_SYS_I2C_OMAP24XX
+
+#undef CONFIG_SPI
+#undef CONFIG_OMAP3_SPI
+#undef CONFIG_CMD_SPI
+
+#undef CONFIG_SYS_LONGHELP
+#undef CONFIG_AUTO_COMPLETE
+#undef CONFIG_CMDLINE_EDITING
+
+#undef CONFIG_FAT_WRITE
+
+#else /* not CONFIG_SPL_BUILD */
+
+#define CONFIG_CMD_ELF
+#define CONFIG_API
+#define CONFIG_SYS_MMC_MAX_DEVICE 1
+
+#endif /* CONFIG_SPL_BUILD */
+
+#undef CONFIG_BOOTCOMMAND
+#define CONFIG_BOOTCOMMAND \
+        "mmc rescan; fatload mmc 0:1 88000000 ubldr; bootelf 88000000"
+
+
 #endif /* __CONFIG_PANDA_H */
-- 
1.9.1

