From 6857bbc8b0c36c8c2837c957f5d4d4bc56fbc6b4 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Tue, 4 Nov 2014 16:40:57 -0500
Subject: [PATCH] freebsd/crochet config changes

---
 include/configs/duovero.h | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/include/configs/duovero.h b/include/configs/duovero.h
index e68f415..b7fd944 100644
--- a/include/configs/duovero.h
+++ b/include/configs/duovero.h
@@ -59,4 +59,17 @@
 
 #define CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG
 
+/* freebsd/crochet build requirements */
+#undef CONFIG_SYS_THUMB_BUILD
+
+#ifndef CONFIG_SPL_BUILD
+#define CONFIG_CMD_ELF
+#define CONFIG_API
+#define CONFIG_SYS_MMC_MAX_DEVICE 1
+#endif
+
+#undef CONFIG_BOOTCOMMAND
+#define CONFIG_BOOTCOMMAND \
+	"mmc rescan; fatload mmc 0:1 88000000 ubldr; bootelf 88000000"
+
 #endif /* __CONFIG_DUOVERO_H */
-- 
1.9.1

